docs/guides/basic-rbac.mdx

```diff
@@ -1,44 +1,44 @@
 ---
-title: Implement basic Role Based Access Control with Metadata
-description: Learn how to leverage Clerk's publicMetadata to implement your own basic Role Based Access Controls
+title: Implement basic Role Based Access Control with metadata
+description: Learn how to leverage Clerk's publicMetadata to implement your own basic Role Based Access Controls.
 ---
 
-# Implement basic Role Based Access Control (RBAC) with Metadata
+# Implement basic Role Based Access Control (RBAC) with metadata 
 
-Clerk provides [User Metadata](/docs/users/metadata#user-metadata), which is a tool that can be leveraged to build flexible custom logic into your application. This guide will cover using Public Metadata to build a based Role Based Access Control (RBAC) system for your application. 
+To control which users can access certain parts of your application, you can leverage Clerk's [roles](/docs/organizations/roles-permissions#roles) feature. Although Clerk offers roles as part of the feature set for [organizations](/docs/organizations/overview), enabling organizations is *not* required to use roles. This guide will cover setting up a basic Role Based Access Control (RBAC) system for products that don't need organizations.
```

Comment:
this can be re-written if it's not very accurate or clear, I kind of wrote this off the top of my head. but the guide definitely needs an explanation to why a user would need to build an RBAC (in order to control which users access certain parts of their app, using Clerk's roles and publicMetadata features)

Reply:
"Although Clerk offers roles as part of the feature set for [organizations](/docs/organizations/overview), enabling organizations is *not* required to use roles."

I think this is a little confusing 'There are roles in orgs...but you don't need orgs for roles.' I get what you mean, but I think clarifying 'virtual' roles as covered in this guide and the roles from Orgs a little more clearly would be perfect.

Reply:
I think this is confusing as well. We have a feature called "roles," which this technique doesn't share. Something like : "Although Clerk offers a roles feature as part of [organizations](/docs/organizations/overview), not every app implements organizations. This guide will cover a workaround for setting up a basic Role Based Access Control (RBAC) system for products that don't use Clerk's organizations or roles."